charts/images.yaml

```diff
@@ -1,20 +1,20 @@
 images:
 - name: terraformer
-  sourceRepository: github.com/gardener/terraformer
-  repository: eu.gcr.io/gardener-project/gardener/terraformer
-  tag: "v1.5.0"
+  sourceRepository: https://github.com/gardener/terraformer
```

Comment:
Hmm, it seems the https://concourse.ci.gardener.cloud/builds/289964 step does not like these `https://` :(

Reply:
Huh. No problem, I can remove those. But is the CI selective about them? Note that *some* of the images have `https://` while others don't:

- https://github.com/gardener/gardener-extension-provider-packet/blob/49af05c8218229b8ab7104244f33cf72cc87ec30/charts/images.yaml#L7
- https://github.com/gardener/gardener-extension-provider-packet/blob/49af05c8218229b8ab7104244f33cf72cc87ec30/charts/images.yaml#L11

I just wanted to tidy this file up. Will undo my changes for now.

Reply:
Actually I removed the `https://` from everything. Let's see if this is better :slightly_smiling_face: